store/postgres/src/relational.rs

```diff
@@ -863,6 +863,13 @@ impl Layout {
         Ok(count)
     }
 
+    pub fn truncate_tables(&self, conn: &PgConnection) -> Result<(StoreEvent, i32), StoreError> {
+        for table in self.tables.values() {
+            conn.execute(&format!("TRUNCATE TABLE {}", table.qualified_name))?;
+        }
+        Ok((StoreEvent::new(vec![]), -1))
```

Comment:
Since we truncate, we know that there will be 0 entities left, so there's no point in returning a count.